python/py-isce2/Portfile

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+PortGroup           active_variants 1.1
+PortGroup           compilers 1.0
+
+github.setup        isce-framework isce2 2.3.1 v
+revision            1
+name                py-isce2
+categories          python
+platforms           darwin
+license             Apache
+
+maintainers         @piyushrpt \
+                    @mazhe \
+                    openmaintainer
```

Comment:
Also, the maintainers here is not correct. You need to include email addresses with the github IDs. (See the documentation or other portfiles for examples.)

Reply:
Will do